mu4e-overview.el

```diff
@@ -197,6 +201,10 @@ passed to CALLBACK will be 0."
                       :unread-count nil
                       :children nil))
                    (mu4e-get-maildirs)))
+         (separators (concat
+                      (if (memq ?- mu4e-overview-maildir-separators)
+                          (cons ?- (delq ?- mu4e-overview-maildir-separators))
```

Comment:
We will be modifying that variable every time we do this because of `delq`. You should use `remq`.
Also can you add a comment here that says why we need to ensure that `-` is the first character? (As I understand it, we're doing that to make sure we don't create a regexp like `[a-z]`)

Reply:
Ups. Fixed.